app/assets/javascripts/components/buttons/form-button-component.js

```diff
@@ -0,0 +1,47 @@
+ManageIQ.angular.app.component('formButtonComponent', {
```

Comment:
Uh.. and since the PR for the buttons component was separated, you should probably remove it from here ;)